app/models/institution.rb

```diff
@@ -159,6 +159,11 @@ class Institution < ActiveRecord::Base
   validates :institution_type_name, inclusion: { in: TYPES }
 
   has_many :yellow_ribbon_programs, dependent: :destroy
+  has_many :school_certifying_officials, dependent: :destroy
+
+  def school_certifying_officials
```

Comment:
What is the purpose this function?

Reply:
https://guides.rubyonrails.org/association_basics.html#options-for-belongs-to-foreign-key
You should be able to use this in `school_certifying_officials` to remove the need for the function

Reply:
Great I appreciate you pointing me in the right direction for the keys there, that makes sense